spec/octokit/client/reviews_spec.rb

```diff
@@ -58,6 +58,17 @@
     end
   end
 
+  describe '.update_pull_request_review' do
```

Comment:
Instead of hardcoding this, can we add a new context "with pull request review" and move this under there?

Reply:
works for me, updated